src/buildHTML.js

```diff
@@ -376,6 +376,7 @@ groupTypes.supsub = function(group, options, prev) {
         supsub.children[1].style.marginRight = scriptspace;
     }
 
+    supsub.classes.push("msupsub");
```

Comment:
We generally prefer doing something like `makeSpan(["msupsub"], [supsub])` instead of mutating elements, does that still work?